### ISSUE TITLE
Ladění zobrazení mapy

### ISSUE THREAD

**Issue by motyc** [ARUP-CAS/aiscr-webamcr#2595]
V aktuální verzi se ukázala potřeba drobných optimalizací mapy:

- [x] Vrstva ZTM má nově kolem sebe bílý rámeček, který vypadá divně. Pokud to jde, změnit základní pozadí okna Leaflet na bílé, aby rámeček nebyl vidět, nebo naopak rámeček odstranit (zprůhlednit).
- [x] Zdá se, že se nějak nesprávně počítá, kdy se mají zapínat jednotlivé typy zobrazení geometrie - clustery - heatmapa. Mělo původně platit:
-- více než 10 000 prvků -> vždy heatmapa
-- 500-10 000 prvků -> clustery
-- < 500 prvků nebo vysoký zoom level -> plné geometrie
Působí to, že prahy jsou nastaveny jinak. Chování přechodu clustery -> geometrie funguje myslím uživatelsky vhodně, problém je zejména přechod clustery -> heatmapa, kdy se heatmapa zapíná velmi brzy. Může to souviset se změnou souřadnicového systému? Prosím analyzovat, jak to teď funguje a případně se domluvit na vhodné úpravě.

**Comment by jhavrlant** [comment]
@motyc Byl tam nastavený limit 5000 pro zobrazení heatmap. Můžu to dát na 10 000, ale zobrazení se zpomalí a nevím jestli to uživateli pomůže, že uvidí dříve clustery.

**Comment by motyc** [comment]
@jhavrlant No, jenže ono to přepne, i když na obrazovce je právě viditelně těch záznamů fakt málo. Zkus to prosím nejdřív pořádně otestovat, pak měň parametry. Přijde mi, že tam spíš bude nějaký renonc.

**Comment by jhavrlant** [comment]
@motyc  Tak to mi budeš muset ukázat nějaké konkrétní místo, mě se to chovalo podle mého očekávání.
